containerd.service

```diff
@@ -18,6 +18,8 @@ Documentation=https://containerd.io
 After=network.target dbus.service
 
 [Service]
+Environment="INOTIFY_INSTANCES_LIMIT=1024"
+ExecStartPre=/bin/sh -c 'current=$(cat /proc/sys/fs/inotify/max_user_instances); [ "$current" -lt ${INOTIFY_INSTANCES_LIMIT} ] && echo ${INOTIFY_INSTANCES_LIMIT} > /proc/sys/fs/inotify/max_user_instances || true'
```

Comment:
Wondering if this is a setting that should be in the default `containerd.service` in this repository, or if it would be better suited for a place in the documentation (and how to add a drop-in/override to custom these settings based on the user's requirements) 🤔

Reply:
I want to find a way like cgroup v1 (unix.Eventfd) but cgroup v2 doesn't support that. or we can let containner create a  Inotify instance, or let shim share  a Inotify instance. set max_user_instances in service is not a good idea.

Reply:
I have seen kubernetes nodes usually have a value of 8192 for max instance. image-builder also uses the same value https://github.com/kubernetes-sigs/image-builder/blob/main/images/capi/ansible/roles/node/tasks/main.yml#L65

Reply:
I find k8s set max_user_instances to 8192  https://github.com/kubernetes/kubernetes/pull/130990/files